stdlib/pdb.pyi

```diff
@@ -168,7 +167,6 @@ class Pdb(Bdb, Cmd):
 def find_function(funcname: str, filename: str) -> tuple[str, str, int] | None: ...
 def main() -> None: ...
 def help() -> None: ...
-def getsourcelines(obj: _SourceObjectType) -> tuple[list[str], int]: ...
```

Comment:
Why are we deleting this entirely? Some users might be depending on this being in the stub. Can't we just put the definition inside an `if sys.version_info < (3, 10)` branch?